### PR TITLE
Updated ReadMe File

### PR DESCRIPTION
**Pull request by pandilladivya**
Updated README.md file.
Added Basic installation steps and introduction to VOLA.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,36 @@
-# volunteers-android
\ No newline at end of file
+
+# Volunteers-Android
+
+## Introduction
+
+Volunteers Android  is a mobile application which helps manage the Hoppers at the Grace Hopper Celebration. We want to provide Systers’s affinity groups a method of collaborating, organizing, and promoting their individual events (i.e. meetups, local workshops, etc). Systers can register, login, and volunteer for events in their local city. These volunteers work multiple shifts, up to eight hours, spread over the three days of the conference. They must commit to their assigned shifts, attend an orientation and be in the area during the conference week.
```
Comment (sandarumk) [inline README.md]: which helps to manage

Comment (sandarumk) [inline README.md]: @MeepyMay We changed the requirement of this being the hopper app to a completely general volunteers app right? I feel like last two lines are solely based on GHC in mind

File: README.md [hunk 2]
```diff
@@ -1 +1,36 @@
...
+
+
+You can learn more about the Systers organization, whose members work on the development of this app [here](http://anitaborg.org/get-involved/systers/) and join our mailing list [here](http://systers.org/mailman/listinfo/systers)
+
```
Comment (sandarumk) [inline README.md]: Add slack too

File: README.md [hunk 3]
```diff
@@ -1 +1,36 @@
...
+## Getting Started
+
+Download [Android Studio](http://developer.android.com/sdk/index.html) and the required SDK tools or get the Eclipse ADT plugin
+
+## Installation: 
+Go to your terminal and execute this command
+
```
Comment (sandarumk) [inline README.md]: If you want to contribute, forking and cloning your own repo makes sense than cloning the systers repo

File: README.md (hunk 2 above)
Comment (pandilladivya) [inline README.md]: Okay

File: README.md [hunk 4]
```diff
@@ -1 +1,61 @@
...
+```
+
+
+## Opening and Running
+
+Now that you have cloned the repo:
+
+1. Launch android studio
+2. Click on File, then New and then Import Project Or from the welcome screen of android studio click on Import project. (If you wish to contribute, you can directly link to GitHub)
+3. Browse to the directory where you cloned (extracted the zip) Volunteers-Android App and take OK
```
Comment (pandilladivya) [inline README.md]: I think Click on Ok would be better. I'll change :)

File: README.md [hunk 5]
```diff
@@ -1 +1,61 @@
...
+
+```
+
+
+## Opening and Running
+
+Now that you have cloned the repo:
+
+1. Launch android studio
+2. Click on File, then New and then Import Project Or from the welcome screen of android studio click on Import project. (If you wish to contribute, you can directly link to GitHub)
```
Comment (pandilladivya) [inline README.md]: Okay. I'll list some reference links :)

File: README.md (hunk 1 above)
Comment (pandilladivya) [inline README.md]: I'll modify that :)

**Comment by sandarumk** [comment]
@pandilladivya can you finish this up?

**Comment by pandilladivya** [comment]
@sandarumk Okay. Will do it as soon as possible.

File: README.md (hunk 5 above)
Comment (pandilladivya) [inline README.md]: @HimanshiJain @sandarumk Should I mention a separate section called References and list some resources like Android docs, github docs, firebase docs etc?

Is that a good idea?

Comment (sandarumk) [inline README.md]: Add these links in the wiki by making a page and link that page to the READ ME

File: README.md [hunk 6]
```diff
@@ -6,21 +6,46 @@
...
+It is a simple two-step process.
+- On GitHub, navigate to the [Volunteers-Android](https://github.com/systers/volunteers-android/) Repository.
+- In the top-right corner of the page, click Fork.
+
+That's it! Now, you have a fork of the original Volunteers-Android repository.
+
+
+### Clone the Repository
+
+You get started by cloning the project to your local machine:
```
Comment (pandilladivya) [inline README.md]: okay.

File: README.md [hunk 7]
```diff
@@ -6,21 +6,46 @@
...
+
+A fork is a copy of a repository.
+
+Forking a repository allows you to freely experiment with changes without affecting the original project.
+
+It is a simple two-step process.
+- On GitHub, navigate to the [Volunteers-Android](https://github.com/systers/volunteers-android/) Repository.
+- In the top-right corner of the page, click Fork.
+
+That's it! Now, you have a fork of the original Volunteers-Android repository.
```
Comment (pandilladivya) [inline README.md]: okay

**Comment by pandilladivya** [comment]
@sandarumk @HimanshiJain 
I've made the suggested changes.
Please review and let me know if there are any.
Thanks 😊

File: README.md [hunk 8]
```diff
@@ -1 +1,65 @@
...
\ No newline at end of file
+
+# Volunteers-Android
+
+## Introduction
+
+Volunteers Android  is a mobile application which helps to manage the Volunteers at an event. We want to provide Systers’s affinity groups a method of collaborating, organizing, and promoting their individual events (i.e. meetups, local workshops, etc). Systers can register, login, and volunteer for events in their local city.
+
+
+You can learn more about the Systers organization, whose members work on the development of this app [here](http://anitaborg.org/get-involved/systers/) , join our mailing list [here](http://systers.org/mailman/listinfo/systers) and you can join our slack group. To get an invitation to the slack group [click here](http://systers.io/slack-systers-opensource/).
```
Comment (sandarumk) [inline README.md]: The information here is inconsistent. For example you have directed users to the systers mailing list and systers open source slack channel. What you can do is tell that this is develop by systers open source team and give the mailing list and slack channel information about systers open source team. And then if needed add additional info about systers and give their mailing list address and slack group.

File: README.md (hunk 5 above)
Comment (HimanshiJain) [inline README.md]: Create an empty page in the wiki/issue for now so that we remember to update this?

Comment (pandilladivya) [inline README.md]: Okay.

File: README.md [hunk 9]
```diff
@@ -1 +1,65 @@
...
+4. Let Android studio import the project, sync, run and build the Gradle.
+5. If Gradle finishes to build without error run the project by clicking on the play button on the tool bar.
+6. Wait for a few seconds and the app should start on your device(android phone or emulator)
+
+
+## Troubleshooting/Issues
+
+If there are any problems encountered when installing the IDE, you may want to check and make sure you have a JDK and JRE installed. If you experience trouble building or running the project, you may not have the necessary build tools, which you can download directly through the IDE. If the app does not run on the emulator, it may be that the emulator is too slow, and you should look into installing one from an outside source.
+
+If there are any other questions or concerns, you can join #vola channel in our slack group. To get an invitation to the slack group [click here](http://systers.io/slack-systers-opensource/).
```
Comment (sandarumk) [inline README.md]: This information is redundant

File: README.md (hunk 8 above)
Comment (pandilladivya) [inline README.md]: Is it clear if we make it this way? @sandarumk @HimanshiJain 


You can learn more about the Systers organization, whose members work on the development of this app [here](http://anitaborg.org/get-involved/systers/).

If you want to join our mailing list [here](http://systers.org/mailman/listinfo/systers) and you can join our slack group.

To get an invitation to the slack group [click here](http://systers.io/slack-systers-opensource/).

Comment (HimanshiJain) [inline README.md]: Or can it be:

Learn more about the Systers organisation and it's open source development team here.
Join the mailing list here. Join our slack group(systers-opensource) or get invitation here.

Does it look better?

Comment (pandilladivya) [inline README.md]: Yes!

**Comment by pandilladivya** [comment]
@HimanshiJain @sandarumk 
Updated the PR.
😊

**Pull request merged**